HW0/P4/P4.py

```diff
@@ -30,33 +30,60 @@
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
     #####################
-
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '--b', label='True trajectory')
+    # load txt file
+    s_true = np.loadtxt("P4_trajectory.txt", delimiter = ",")
+    ax.plot(s_true[:,0],s_true[:,1] , s_true[:,2] ,'--b', label='True trajectory')
 
     #####################
     # Part 2:
     #
     # Read the observation array and plot it (Part 2)
     #####################
-
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    s_measure = np.loadtxt("P4_measurements.txt", delimiter = ",")
+    # construct the streching matrix
+    s_streching = np.matrix([[1/rx,0,0], [0,1/ry,0],[0,0,1/rz]])
```

Comment:
Another option would be to use np.diag(np.array()), rather than write out all those zeroes.